client/containers/DashboardSubmissions/ArbitrationMenu.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Button } from '@blueprintjs/core';
+
+import { DefinitelyHas, Pub, SubmissionStatus, DocJson } from 'types';
+import { Icon, IconName, DialogLauncher } from 'client/components';
+import { apiFetch } from 'client/utils/apiFetch';
+import VerdictDialog from './VerdictDialog';
+
+require('./arbitrationMenu.scss');
+
+type Props = {
+	pub: DefinitelyHas<Pub, 'submission'>;
+	onJudgePub: (pubId: string, status?: SubmissionStatus) => void;
+};
+
+const ArbitrationMenu = (props: Props) => (
+	<div className="arbitration-menu-component">
+		<div style={{ gridColumn: 1 }}>
+			<DialogLauncher
+				renderLauncherElement={({ openDialog }) => (
+					<Button
+						minimal
+						small
+						icon={<Icon icon={'cross' as IconName} iconSize={20} />}
+						onClick={openDialog}
+					/>
+				)}
+			>
+				{({ isOpen, onClose }) => (
+					<VerdictDialog
```

Comment:
You might find a `ConfirmDialog` helpful here if you don't want to squeeze this functionality into the `VerdictDialog`. Either way.